tests/Microsoft.Identity.Test.Unit.net45/Resources/ExpectedTokenCache.json

```diff
@@ -0,0 +1,201 @@
+﻿{
```

Comment:
@rayluo - check out this serialized cache that contains both app metadata and an FRT. Let me know if the format isn't what you expect.